KarhooUISDK/Screens/Rides/RidesList/RideDetails/KarhooRideDetailsPresenter.swift

```diff
@@ -22,6 +22,7 @@ final class KarhooRideDetailsPresenter: RideDetailsPresenter {
     private let analyticsService: AnalyticsService
     private let tripFeedbackScreenBuilder: TripFeedbackScreenBuilder
     private let tripRatingCache: TripRatingCache
+    private var alertHandler: AlertHandlerProtocol?
```

Comment:
Do we need the alertHandler ? does not seem to be used in this class

Reply:
Nope. Deleted.